packages/@react-spectrum/toast/src/ToastContainer.tsx

```diff
@@ -141,8 +135,7 @@ export function ToastContainer(props: SpectrumToastContainerProps): ReactElement
                 key={toast.key}
                 className={classNames(toastContainerStyles, 'spectrum-ToastContainer-listitem')}
                 style={{
-                  // @ts-expect-error
-                  viewTransitionName: `_${toast.key.slice(2)}`,
```

Comment:
Moved this hack into the hooks themselves to `toast.key` is always a valid CSS identifier.